src/styles/sidebar/components/thread.scss

```diff
@@ -5,28 +5,35 @@
 
   &--reply {
     margin-top: 0.5em;
-    padding-top: 0.5em;
   }
 
+  // Conserve space for deeper usable nesting
```

Comment:
The changes in this SCSS module are primarily because I moved the thread border (dashed line) from `border-left` to `border-right`.